inputs/flexibility/energy/capacity_of_energy_heat_flexibility_p2h_heatpump_electricity.ad

```diff
@@ -12,7 +12,7 @@
       )
     )
 - priority = 0
-- max_value_gql = present:DIVIDE(DIVIDE(MAX(0.5 * Q(heat_production_total) * BILLIONS, V(energy_heat_flexibility_p2h_heatpump_ht_electricity, output_of_steam_hot_water)*2), V(energy_heat_flexibility_p2h_heatpump_ht_electricity,full_load_hours)), MJ_PER_MWH)
+- max_value_gql = present:DIVIDE(DIVIDE(MAX(3 * Q(heat_production_total) * BILLIONS, V(energy_heat_flexibility_p2h_heatpump_ht_electricity, output_of_steam_hot_water)*12), V(energy_heat_flexibility_p2h_heatpump_ht_electricity,full_load_hours)), MJ_PER_MWH)
```

Comment:
Same comment as above, but then for the capacity_of_energy_heat_flexibility_p2h_boiler_electricity.

Reply:
I have updated `capacity_of_energy_heat_flexibility_p2h_boiler_electricity` by a factor of 6 as well.